.ccache/ccache.conf

```diff
@@ -0,0 +1,32 @@
+# For complete documentation, see https://ccache.dev/manual/4.6.1.html#_configuration
+
+# This option specifies the maximum size of the cache. Use 0 for no limit. The default value is 5G.
+# Available suffixes: k, M, G, T (decimal) and Ki, Mi, Gi, Ti (binary). The default suffix is G
+max_size = 10G
+
+# This option specifies where ccache will keep its cached compiler outputs.
+cache_dir = /mnt/ccache/
+
+#If true, ccache will compress data it puts in the cache. However, this option has no effect on how files are
+#retrieved from the cache; compressed and uncompressed results will still be usable regardless of this option.
+# The default is true.
+compression = true
+
+# This option determines the level at which ccache will compress object files using the real-time compression
+# algorithm Zstandard. It only has an effect if compression is enabled (which it is by default). In restricted space environments,
+# it is recommended to set the value of this parameter to 19 (or via environment parameter CCACHE_COMPRESSLEVEL)
+compression_level = 1
+
+# When true, ccache will just call the real compiler, bypassing the cache completely. The default is false.
+disable = false
+
+# If set to a file path, ccache will write information on what it is doing to the specified file.
+# This is useful for tracking down problems.
+log_file = /mnt/ccache/ccache.log
```

Comment:
Should probably be local

Reply:
I'm going to move both log files into concord-bft/build/ccache/

Reply:
Hi, I've decided to remove this log completely for now for multiple reasons. In case we need to debug i'll add it.